pkg/partmgr/partmgr_test.go

```diff
@@ -72,14 +72,18 @@ func TestCreateNewPartition(tst *testing.T) {
 }
 
 func getPartitionManager(tst *testing.T) *PartitionManager {
-	schema := testutils.CreateSchema(tst, "*")
+	schm, err := schema.NewSchema([]string{"*"})
```

Comment:
You could just left it as is, and let the testutil.CreateSchema deal with errors and test fails.